security/compliance_operator/compliance-operator-installation.adoc

```diff
@@ -0,0 +1,140 @@
+[id="compliance-operator-installation"]
+= Installing the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-operator-installation
+
+toc::[]
+
+
+Before you can use the Operator, you must ensure it is deployed in the cluster.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+. Create the target namespace:
++
+[source,terminal]
+----
+$ oc create -f deploy/ns.yaml
```

Comment:
Let's change this to:

```bash
$ cat <<EOF > oc apply -f -
apiVersion: v1
kind: Namespace
metadata:
  name: openshift-compliance
EOF
```

or:

namespace.yaml:

```yaml
---
apiVersion: v1
kind: Namespace
metadata:
  name: openshift-compliance
```
then do:

```bash
$ oc apply -f namespace.yaml
```

this way, we don't need to point customers to the upstream repo and instead they can just do this from the docs directly. They can also do this from the UI.